src/transformers/models/gpt2/modeling_tf_gpt2.py

```diff
@@ -739,6 +713,7 @@ def __init__(self, config, *inputs, **kwargs):
         output_type=TFBaseModelOutputWithPastAndCrossAttentions,
         config_class=_CONFIG_FOR_DOC,
     )
+    @unpack_inputs
```

Comment:
In other related PRs, `@unpack_inputs` is added as the top decorator. Would you mind moving to the top as well? 🙏  (there are other similar instances below)

Reply:
sure thing; will ping you when it's done

Reply:
✔️